lib/que/job.rb

```diff
@@ -94,13 +94,17 @@ def enqueue(*args)
           end
         end
 
+        tags = job_options[:tags] || []
+        data = { tags: tags }.merge(job_options[:data] || {})
+        # TODO: some protection about overall data size
```

Comment:
Do we need protection for this? If people want to store arbitrarily large data here it should be allowed right? Since important indexes like `que_poll_idx` and `que_jobs_args_gin_idx` don't include this column, large data shouldn't have too much of an impact on performance.